pages/rest-api/start.mdx

```diff
@@ -5,7 +5,7 @@ import { Tab, Tabs,Callout } from "nextra-theme-docs";
 ## Base URL
 
 ```
-https://waifu.it/api
+https://waifu.it/api/v4
```

Comment:
Keep the base url /api, remove the changes

Reply:
Alright :)